charts/library/common/values.yaml

```diff
@@ -1162,6 +1162,10 @@ cnpg:
       # Additional Labels and annotations for cnpg cluster
       labels: {}
       annotations: {}
+      # Additional environment variables to set on all pods created in the cluster
+      env: {}
+      envFrom: {}
+      envList: {}
```

Comment:
lets not support envList on cnpg, tbh it should be removed from container are well at some point.

@PrivatePuffin maybe on the next breaking change?